pkgs/servers/http/nginx/nix-skip-check-logs-path.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/auto/install b/auto/install
+index d884487..dccc411 100644
+--- a/auto/install
++++ b/auto/install
+@@ -148,12 +148,6 @@ install:	build $NGX_INSTALL_PERL_MODULES
+ 		|| cp conf/nginx.conf '\$(DESTDIR)$NGX_CONF_PATH'
+ 	cp conf/nginx.conf '\$(DESTDIR)$NGX_CONF_PREFIX/nginx.conf.default'
+ 
+-	test -d '\$(DESTDIR)`dirname "$NGX_PID_PATH"`' \\
```

Comment:
usually, a non-empty `DESTDIR` suggests you're not installing to the target system, but into a fakeroot.

Most build systems skip "locally setting up things" if `DESTDIR` is set to something non-empty (https://github.com/systemd/systemd/blob/878f2dae77e61ee6765b72318f49f8226f45ec0f/catalog/meson.build#L33), or allow skipping setting up outside directories (https://github.com/systemd/systemd/blob/52d8bcd131e7f3c9b7629412302e1ea46ddb70ca/src/journal/meson.build#L114).

I'm not too familiar with nginx's build system, but there might be a way to upstream these changes in a generic fashion?

Reply:
Did not find another variant